source/iterators.tex

```diff
@@ -6520,6 +6523,28 @@
 \pnum \returns \tcode{N}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{ssize(C\& c)}}%
+\begin{itemdecl}
+template <class C>
+constexpr auto ssize(const C& c)
```

Comment:
Place on same line so it appears as it does in the declaration in the synopsis.

Reply:
Fixed.